robottelo/hosts.py

```diff
@@ -1118,6 +1118,28 @@ def install_tracer(self):
             raise ContentHostError('There was an error installing katello-host-tools-tracer')
         self.execute('katello-tracer-upload')
 
+    def register_to_cdn(self):
```

Comment:
@adarshdubey-star this is relevant to your work for CloudConnector as well - thanks to @Gauravtalreja1 we've got this method that you can build off of in order to register within the test setup or test execution (against stage or production)